service/utils/post_utils.js

```diff
@@ -7,6 +7,10 @@ export function isSystemMessage(post) {
     return post.type && post.type.startsWith(Constants.SYSTEM_MESSAGE_PREFIX);
 }
 
+export function shouldIgnorePost(post) {
+    return Constants.IGNORE_POST_TYPES.indexOf(post.type) !== -1;
```

Comment:
I find it more readable and semantic to use `Array.includes` instead of `indexOf !== -1`.
```
return Constants.IGNORE_POST_TYPES.includes(post.type);
```